### ISSUE TITLE
Issues with loading extension

### ISSUE THREAD

**Issue by charleskawczynski** [timholy/SnoopCompile.jl#400]
I've never ran into this before, but I'm having trouble loading the extension in SnoopCompile:

```julia
julia> using SnoopCompileCore

julia> tinf = @snoop_inference begin
           import Thermodynamics as TD
       end
SnoopCompileCore.InferenceTimingNode(SnoopCompileCore.InferenceTiming(InferenceFrameInfo for Core.Compiler.Timings.ROOT(), 0.335086625, 0.335086625), 747264.775509541, SnoopCompileCore.InferenceTimingNode[], nothing, #undef)

julia> using PyPlot

julia> import PyPlot

julia> using Profile

julia> using SnoopCompile

julia> using SnoopCompile: pgdsgui

julia> mref, ax = pgdsgui(tinf);
ERROR: MethodError: no method matching pgdsgui(::SnoopCompileCore.InferenceTimingNode)
Stacktrace:
 [1] top-level scope
   @ REPL[8]:1
```

If I grab the extension directly, it seems like it has loaded:

```julia
julia> SCPyPlotExt = Base.get_extension(SnoopCompile, :SCPyPlotExt);

julia> SCPyPlotExt.pgdsgui
pgdsgui (generic function with 3 methods)
```

But then I still run into a strange (seemingly loading) issue when trying to use it:
<img width="1614" alt="Screenshot 2024-10-15 at 9 31 51 AM" src="https://github.com/user-attachments/assets/89cac42d-ef70-4361-9d49-162482517309">

Any idea what I'm doing wrong here?

**Comment by charleskawczynski** [comment]
Ah, I think `pgdsgui` is just not getting imported in the extension. I can open a PR.

**Comment by charleskawczynski** [comment]
@sbozzolo beat me to it! Will be closed by https://github.com/timholy/SnoopCompile.jl/pull/398.

**Comment by timholy** [comment]
Closed by #398
